run.py

```diff
@@ -170,6 +170,51 @@ def reindex(since=None, batch_size=None, force_indexing=False, update_solr=True,
         raise e
 
 
+def rebuild_collection(collection_name):
+    """
+    Will grab all recs from the database and send them to solr
+    """
+    
+    now = get_date()
+    if collection_name.startswith('http'):
+        solr_urls = [collection_name]
+    else:
+        solr_urls = []
+        urls = app.conf['SOLR_URLS']
+        for u in urls:
+            parts = u.split('/')
+            parts[-2] = collection_name
+            solr_urls.append('/'.join(parts))
+    
+    logger.info('Sending all records to: %s', ';'.join(solr_urls))
+    sent = 0
+
+    batch = []
+    with app.session_scope() as session:
+        # deleted docs are in the db, no? but i don't see how to filter them out
+        for rec in session.query(Records) \
+            .options(load_only(Records.bibcode, Records.updated, Records.processed)) \
+            .yield_per(1000):
+
+            sent += 1
+            if sent % 1000 == 0:
+                logger.debug('Sending %s records', sent)
+
+            batch.append(rec.bibcode)
+            if len(batch) > 1000:
+                tasks.task_index_records(batch, force=True, update_solr=True,
+                                           update_metrics=False, update_links=False,
+                                           ignore_checksums=True, solr_targets=solr_urls)
+                batch = []
+
+    if len(batch) > 0:
+        tasks.task_index_records(batch, force=True, update_solr=True,
+                                           update_metrics=False, update_links=False,
+                                           ignore_checksums=True, solr_targets=solr_urls)
+        
+    logger.info('Done processing %s records', sent)
```

Comment:
a small thing, but we change the log message to 'Done rebuilding collection, sent %s records' to support easy kibana queries?